section2/exercises/if-statements.js

```diff
@@ -65,9 +72,11 @@ When numQuarters = 3, program should log "I have enough money for a gumball"
 var numQuarters = 0;
 
 // Write your conditional here
+if (numQuarters >= 2) {
 console.log("I have enough money for a gumball");
+} else if (numQuarters < 2) {
```

Comment:
You could just use `else` here without checking another condition since any number of quarters less than 2 won't pass the first condition.